.clang-format-ignore

```diff
@@ -1,9 +1,12 @@
 # C++ code generated from Slice files
-cpp/include/generated
-cpp/src/**/generated
-cpp/test/**/generated
-cpp/src/**/msbuild/*
-cpp/test/**/msbuild/*
+cpp/include/generated/**/*.h
```

Comment:
The previous version didn't work, and reformatting the generated code messed up the NOLINT comments.

Reply:
Interesting, we've been using this with CI for a while.